src/css/index.scss

```diff
@@ -12,6 +12,5 @@ $font-size-xs: 0.75rem; // 12px
 $icon-font: FontAwesome;
 
 @import "~font-awesome/css/font-awesome.css";
-@import "customTable";
 @import "leafletCluster";
 @import "maps";
```

Comment:
yay removal. I think you can even remove leafletCluster and amps since they are not used in bluesquare-components